src/pages/map.tsx

```diff
@@ -155,25 +119,55 @@ const MapPage = ({ schedule }: Props) => {
           height: '100vh',
           width: '100%'
         }}
+        onDrag={() => setActiveLoc(undefined)}
         onStyleLoad={loadedMap => {
           setMap(loadedMap);
           loadedMap.setCenter({ lat: -6.163088, lng: 106.836715 });
         }}
         style="mapbox://styles/mapbox/streets-v8"
       >
-        {coordinates.map((coordinate, i) => {
-          return (
-            // @ts-expect-error - `coordinate` is still untyped
-            <Marker key={i} coordinates={coordinate}>
-              <Mark key={i} lokasi={coordinate.lokasi} />
-            </Marker>
-          );
-        })}
+        <>
+          {coordinates.map((coordinate, i) => {
+            return (
+              //@ts-expect-error - coordinate type conflict
+              <Marker key={i} coordinates={coordinate}>
+                <Box
+                  onClick={() => {
+                    setActiveLoc(coordinate.lokasi);
+                    if (map != null) {
+                      map.easeTo({
+                        //@ts-expect-error latlng conflict
+                        center: {
+                          lat: coordinate.lokasi.lat,
+                          lng: coordinate.lokasi.lon
+                        }
+                      });
+                    }
+                  }}
+                >
+                  <Mark key={i} />
+                </Box>
+              </Marker>
+            );
+          })}
+          {activeLoc != undefined && (
+            <Popup
+              key={activeLoc.osm_id}
+              anchor="bottom"
+              //@ts-expect-error latlng conflict
+              coordinates={{ lat: activeLoc.lat, lng: activeLoc.lon }}
+              style={{ marginTop: -20, padding: 0 }}
+            >
+              <Box bg={colorMode === 'dark' ? 'black' : 'white'}>
+                <VaxLocation isUserLocationExist={false} loading={false} location={activeLoc.parent} />
+              </Box>
+            </Popup>
+          )}
+        </>
       </Map>
-      <Box height={80} left={0} maxWidth={450} position="fixed" top={0} width="100%" zIndex={999999999999}>
-        <Box bg="black" borderRadius={10} margin={2} padding={2}>
+      <Box height="80px" left={0} maxWidth="450px" position="fixed" top={0} width="100%" zIndex={999999}>
+        <Box bg={colorMode === 'dark' ? 'black' : 'white'} borderRadius={10} margin={2} padding={2}>
```

Comment:
```suggestion
        <Box backgroundColor={mapFlyoutBackgroundColor} borderRadius={10} margin={2} padding={2}>
```

Idem. Also use `background-color` instead of `background`.